src/esp/assets/FRLInstanceMeshData.h

```diff
@@ -43,10 +50,19 @@ class FRLInstanceMeshData : public GenericInstanceMeshData {
 
   virtual Magnum::GL::Mesh* getMagnumGLMesh() override;
 
+  const std::vector<vec3f> get_vbo();
+  const std::vector<int> get_ibo();
+
  protected:
   std::vector<vec4f> cpu_vbo;
   std::vector<vec3uc> cpu_cbo;
 
+  std::vector<vec3f>* cpu_vbo_3   = nullptr;
```

Comment:
We name class members with and `_`, i.e. `cpu_vbo_3_` as it makes the implementation code more readable (I see that that wasn't followed for the existing implementation here, oops).